app/controllers/editorial/submissions_controller.rb

```diff
@@ -9,6 +9,12 @@ class Editorial::SubmissionsController < Editorial::SectionsController
     before_action :find_submission, only: [:show, :update]
     decorates_assigned :submission
 
+    def show
+      super
+      # FIXME: this works around the node Fabricator not setting the type in specs
```

Comment:
The issue is not that Fabricator doesn't support `type`. It's that loads of our specs need fixing - the majority are written against `Node`, not `Node` subclasses. When I set the type to `GeneralContent` by default in the Fabricator, loads of specs break.